pkg/monitor/event.go

```diff
@@ -18,8 +18,10 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var ReMatchFirstQuote = regexp.MustCompile(`"([^"]+)"( in (\d+(\.\d+)?(s|ms)$))?`)
```

Comment:
Looks duplicated at the package level and then redeclared below in startEventMonitoring?

Reply:
I was trying to re-use it and neglected to remove it from the startEventMonitoring function.